src/codeu/chat/client/commandline/Chat.java

```diff
@@ -423,6 +432,72 @@ private ConversationContext find(String title) {
 				return null;
 			}
 		});
+		
+		
+		// Interest commands
+		// 
+		// Add
+		panel.register("i-add-user", new Panel.Command() {
+			public void invoke(List<String> args) {
+				// append the name by space from args and send to usercontext
+				String name = !args.isEmpty() ? String.join(" ", args).trim() : "";
```

Comment:
Yes it could, I used Julia's method of combining the args by spaces to account for that.